<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Antibiotic Resistance</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Google Fonts -->
  <link
    href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Montserrat:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i"
    rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">

  <!-- Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

</head>

<body>
  <!-- ======= Welcome Section ======= -->
  <section id="welcome">
    <div class="welcome-container">
      <h1 data-aos="zoom-in">Antibiotic Resistance in E. Coli</h1>
      <h2 data-aos="fade-up">A Genome-wide study on Antibiotic Resistance in E. Coli</h2>
      <h2 data-aos="fade-up">
        <p>Alan Chen &emsp; Myra Haider &emsp; Jiayi Wu</p>
      </h2>
      <a data-aos="fade-up" data-aos-delay="200" href="#about" class="btn-get-started scrollto">Learn more</a>
    </div>
  </section><!-- End welcome -->

  <!-- ======= Header ======= -->
  <header id="header" class="d-flex align-items-center">
    <div class="container d-flex align-items-center justify-content-between">

      <nav id="navbar" class="navbar">
        <ul>
          <li><a class="nav-link scrollto active" href="#welcome">Home</a></li>
          <li><a class="nav-link scrollto" href="#about">About</a></li>
          <li><a class="nav-link scrollto" href="#methodology">Methodology</a></li>
          <li><a class="nav-link scrollto" href="#results">Results</a></li>
          <li><a class="nav-link scrollto" href="#conclusion">Conclusion</a></li>
          <li><a class="nav-link scrollto" href="#faq">Sources</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->

  <main id="main">

    <!-- ======= About Section ======= -->
    <section id="about" class="about">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Abstract</h2>
          <p> One of the greatest challenges in public health is the <b>growing number of bacterial species</b> that
            have <b>developed resistance to antibiotics</b> through point mutations in their genome.
            Our project aims to identify these genetic markers of antibiotic resistance through a genome wide
            association study between 36 antibiotic resistant E.Coli samples and 36 control samples. Genetic variants
            will be checked for statistical significance, and analyzed for any functional effects.
          </p>
        </div>

        <div class="row">
          <div class="col-lg-6" data-aos="fade-right">
            <div class="image">
              <img src="assets/img/antibiotic.png" class="img-fluid" alt="antibiotic resistance">
            </div>
          </div>
          <div class="col-lg-6" data-aos="fade-left">
            <div class="content pt-4 pt-lg-0 pl-0 pl-lg-3 ">
              <h3>"By 2050, superbugs could kill 10 million people a year"</h3>
              <p class="font-italic">
                Although <b>antibiotic resistant bacteria</b> have been studied for decades, their whole genome
                sequencing has started relatively recently <sup>[3]</sup>. Due to their immunity to known
                medicine, their exposure to humans can lead to infections that are currently <b>impossible to cure</b>
                <sup>[1]</sup>.
                According to the International Federation of Pharmaceutical
                Manufacturers & Associations
                (IFPMA), antibiotic resistance will:
              </p>
              <ul>
                <li><i class="bx bx-check-double"></i> <b>Force 28 million more people into poverty<sup>[2]</sup> </b>
                </li>
                <li><i class="bx bx-check-double"></i> <b>Cost the global economy up to 100 trillion
                    dollars<sup>[2]</sup></b></li>
                <li><i class="bx bx-check-double"></i> <b>Kill up to 10 million people a year by 2050<sup>[2]</sup></b>
                </li>
              </ul>
              <p>
                This study focuses on <b>Enterobacteriaceae</b>, specifically E. Coli, a common cause of infections,
                both
                in healthcare
                settings and communities <sup>[4]</sup>. Certain strains of E. Coli have developed an especially
                dangerous
                resistance mechanism, the ability to produce an enzyme known as <b>extended-spectrum
                  beta-lactamase</b>, or ESBL <sup>[4]</sup>. ESBL is capable of breaking down multiple types of
                antibiotics
                such as penicillin, <b>rendering them ineffective</b> <sup>[4]</sup>.
              </p>
              <p>
                Understanding the genetic markers of antibiotic resistance can lead to better, <b>more effective
                  treatments</b> to combat the growing concern of superbugs.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section><!-- End About Us Section -->

    <!-- ======= Methodology Section ======= -->
    <section id="methodology" class="methodology">
      <div class="container">
        <div class="section-title" data-aos="fade-up">
          <h2>Metholodgy</h2>
        </div>
        <div class="row">
          <h4 id="pipeline">An Overview of the Pipeline</h4>

          <figure>
            <img src="assets/img/pipeline.png" class="img-fluid" alt="pipeline">
          </figure>
          <h3>The Data</h3>
          <p class="font-italic">
            The study was conducted using <b>two groups of samples</b>, one with an ESBL-enzyme
            producing strain of E.Coli (denoted with DRR as the suffix), and a non-antibiotic resistant control
            group <sup>[5][6]</sup>. For each group, <b>sample sizes were 36 whole genomes</b>. The
            samples were obtained from <a
              href="https://www.ncbi.nlm.nih.gov/Traces/study/?acc=SRP058582&f=librarylayout_s%3An%3Apaired%3Bbases_l%3An%3A0%253A999999999%3Binstrument_s%3An%3Aillumina%2520miseq%3Ac&o=acc_s%3Aa&s=SRR10001251,SRR10018351,SRR10001259,SRR10001268,SRR10012673,SRR10012674,SRR10012676,SRR10012677,SRR10012698,SRR10012702,SRR10012704,SRR10012906,SRR10012913,SRR10013003,SRR10013021,SRR10013010,SRR10013022,SRR10013025,SRR10013029,SRR10013030,SRR10013032,SRR10013114,SRR10013117,SRR10013127,SRR10013130,SRR10013322,SRR10013157,SRR10013348,SRR10013353,SRR10013371,SRR10013397,SRR10013400,SRR10013405,SRR10013432,SRR10013439,SRR10013442"
              target="_blank"> Illumina NCBI's SRA Run Selector</a> and formatted as <b>.fastq files</b>
            <sup>[5][6]</sup>. Each sample has two
            files associated with it, as they are paired-end reads, totalling to 144 files.
          </p>
          <h3>Quality Control</h3>
          <p>To ensure these 144 files will yield valid results later in the analysis, each was put through a quality
            control software called FastQC. <b>FastQC</b> takes in a single .fastq file and <b>outputs a summary
              report</b> listing
            basic statistics about the reads, including quality scores from base calling, the presence of adapter
            sequences, and percent N content which indicates missing data. </p>
          <p>While it’s important to check for quality of
            each sample individually, the FastQC results of our <b>case and control groups were also aggregated</b> to
            identify any major <b>differences in quality</b> between the two that could affect the <a
              href="https://www.genome.gov/about-genomics/fact-sheets/Genome-Wide-Association-Studies-Fact-Sheet"
              target="_blank">Genome-Wide Association Studies (GWAS)</a> results.
            <b>All 144
              files passed</b> the default FastQC check for basic statistics and adapter content, thus they were all
            included in the study.
            However, the <b>antibiotic resistance group (ESBL) </b> had more samples that had an <b>unusually high per
              sequence GC content</b>, indicated by the higher number of ESBL_Samples that failed this check (see <a
              href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html"
              target="_blank">GC
            </a> and <a
              href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/10%20Adapter%20Content.html"
              target="_blank">Adapter Content</a>). This may either be a <b>physiological phenomenon</b> or a
            <b>confounding factor</b> that may need to be addressed.
          </p>
          <p>During the sequencing process, <b>temporary adapter sequences</b> of nucleotides are attached to the
            fragments of
            DNA in order to <b>facilitate sequencing</b> <sup>[7]</sup>. Sometimes these adapters are accidentally
            sequenced as part of
            the sample genome, so they <b>need to be algorithmically removed</b> using tools such as Cutadapt. <a
              href="https://cutadapt.readthedocs.io/en/stable/" target="_blank">Cutadapt</a>
            was run on all 72 samples using default parameters, in trimmed pair-end reads mode to “remove adapter
            sequences… from high-throughput sequencing reads”<sup>[8]</sup>. The trimmed fastq files were piped through
            FastQC
            again to ensure that the data still passes our <b>quality check post-adapter trimming</b>, which every file
            did
            <sup>[7]</sup>.
          </p>
          <div class="col-lg-6" data-aos="fade-right">

            <div class="tab-content">
              <div class="tab-pane active show" id="tab-1">
                <figure>
                  <img src="assets\img\quality score.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-2">
                <figure>
                  <img src="assets/img/sampleReport.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-3">
                <figure>
                  <img src="assets/img/basicstats.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-4">
                <figure>
                  <img src="assets/img/perGCadapter.png" alt="" class="img-fluid">
                </figure>
              </div>
            </div>
          </div>
          <div class="col-lg-6 mt-4 mt-lg-0" data-aos="fade-left">
            <ul class="nav nav-tabs flex-column">
              <li class="nav-item">
                <a class="nav-link active show" data-bs-toggle="tab" href="#tab-1">
                  <h4>Quality Score</h4>
                  <p>The distribution of quality scores across reads of different lengths</p>
                </a>
              </li>
              <li class="nav-item mt-2">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-2">
                  <h4>Summary Report</h4>
                  <p>Table of summary statistics for a sample</p>
                </a>
              </li>
              <li class="nav-item mt-3">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-3">
                  <h4>Basic Statistics</h4>
                  <p>The barchart depicts the total number of samples that passed the basic statistics test between
                    groups</p>
                </a>
              </li>
              <li class="nav-item mt-4">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-4">
                  <h4>GC and Adapter Content</h4>
                  <p>The barchart depicts the total number of samples that passed the basic statistics test between
                    groups</p>
                </a>
              </li>
            </ul>
          </div>
          <h3>Sequence Alignment, File Conversion, Sorting</h3>
          <p class="font-italic">
            A pair of .fastq files for one <b>sample contains the raw reads</b> from the entire genome that <b>need to
              be put
              together into a contiguous sequence</b>. This is where the process of
            alignment comes in. Using a <b>previously known E. Coli reference genome</b>, we utilized an alignment
            software
            called
            <a href="http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#getting-started-with-bowtie-2-lambda-phage-example"
              target="_blank">Bowtie2</a> to algorithmically pieces together these reads. The
            outputs were <b>36 files
              in sam format</b>, a text readable version of this sequence, <b>per group</b> (72 total). Using <a
              href="http://www.htslib.org/" target="_blank">samtools</a> and <a
              href="https://broadinstitute.github.io/picard/" target="_blank">picard</a>, these sam files were then
            <b>converted to
              .bam files</b> - a compressed version that <b>runs more efficiently</b> through bioinformatics software.
          </p>
          <figure>
            <img src="assets/img/alignment.png" alt="" class="img-fluid">
          </figure>
          <h3>Variant Calling</h3>
          <p class="font-italic">
            The <b>72 cleaned and formatted .bam files</b> then undergo variant calling, the <b>process of
              identifying</b>
            the <a
              href="https://isogg.org/wiki/Single-nucleotide_polymorphism#:~:text=A%20single%2Dnucleotide%20polymorphism%20(SNP,paired%20chromosomes%20in%20an%20individual."
              target="_blank">Single Nucleotide Polymorphisms</a> (SNP's) in each sample. These are the <b>point
              mutations</b> that can
            differentiate samples, with the goal being able to find the <b>SNP’s that are significantly different</b>
            between
            our antibiotic-resistant and control groups. This was done using GATK’s <a
              href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller"
              target="_blank">HaplotypeCaller</a> tool, which <b>outputs
              a vcf</b> that contains the <b>list of variants</b> found in a single sample. All of these vcf files were
            then merged
            into a single file using <a href="http://samtools.github.io/bcftools/bcftools.html"
              target="_blank">bcftools</a> to be <b>applied for our GWAS</b>.
          </p>
          <figure>
            <img src="assets/img/variant.png" alt="" class="img-fluid" style="padding-bottom:30px">
          </figure>
        </div>

      </div>
    </section><!-- End data Section -->

    <!-- ======= Results Section ======= -->
    <section id="results" class="methodology">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Results</h2>
        </div>

        <div class="row">
          <h3>PCA Analysis</h3>
          <p class="font-italic">A basic <a
              href="https://en.wikipedia.org/wiki/Principal_component_analysis#:~:text=Principal%20component%20analysis%20(PCA)%20is,components%20and%20ignoring%20the%20rest."
              target="_blank">Principal component analysis (PCA)</a> was ran <b>to determine if there were any
              meaningful difference</b> between
            the Antibiotic-resistant and Control groups. The principle components were calculated using <a
              href="http://zzz.bwh.harvard.edu/plink/" target="_blank">Plink</a> and the <b>initial results look
              promising</b>, with most of
            the antibiotic-resistant E. Coli samples clustered together on the right. The control samples seem to
            have more variance, especially the three outliers in the top left. Due to our <b>small dataset</b>,
            the <b>outliers were retained</b> in the study.
          </p>
          <figure>
            <img src="assets/img/pca.png" alt="" class="img-fluid">
          </figure>
          <h3>Genome Wide Association Study</h3>
          <p class="font-italic">In order to perform the GWAS, we used a <b>Logistic Regression model</b>, comparing the
            two groups and <b>calculating a p-value for each SNP</b>. The model was further tuned by utilizing the
            <b>first
              two
              principal components as covariates</b>, and removing all <b>SNP’s with a genotyping rate of less than
              60%</b>.
            The genotyping rate is used as a measure of missing data, and we found that 60% was the best
            threshold to <b>minimize the skew</b> of our results.

          </p>
          <figure>
            <img src="assets/img/gwas.png" alt="" class="img-fluid">
          </figure>
          <h3>Variant Annotation</h3>
          <p class="font-italic">To perform <b>further analysis on our variants</b>, we chose <a
              href="https://pcingola.github.io/SnpEff/" target="_blank">snpEff</a> to assist us with
            variant annotation. SnpEff provided us with an E. Coli <b>database to use as the reference</b> in our
            annotation, giving
            us an annotated vcf file, a succinct summary report, and gene_name table. We can
            then use <a href="https://pcingola.github.io/SnpEff/" target="_blank">snpSift</a> to <b>filter out the
              common variants</b> present in both group, <b>leaving only the necessary variants</b>
            . This <b>reduces the scope significantly</b> and enables us to do <b>more specific analyzation</b>
            with it.

          </p>
          <figure>
            <img src="assets/img/annotation.png" alt="" class="img-fluid">
          </figure>
          <h3>snpEff Summary</h3>
          <p class="font-italic">Since the <b>number of errors</b> we
            encountered is relatively <b>small compared to our datasize</b>, we decided to focus more on the <b>numerous
              warnings</b>. These warnings mainly consist of ref_dose_not_match_genome error. We theorized that it may
            be a
            <b>result of using two different reference genomes</b>: the original reference genome and the one provided
            by
            snpEff. It was <b>unavoidable in this study</b> as our initial reference genome was missing an essential
            .gff file.
            Our results could significantly improve if we built our own database. Another observation is that the
            <b>variant
              rate of the esbl group is much higher</b> than the control group. We hypothesize that it is due to the
            possibility
            that the ESBL group is more prone to mutations.


          </p>
          <figure>
            <img src="assets/img/snpeff.png" alt="" class="img-fluid">
          </figure>

          <h3>Overview of Variants</h3>
          <p class="font-italic">Below is brief overview of the <b>base change and variants type</b> of our two groups.
            Base change is a basic gene level mutation which <b>alters the original gene base</b> of a DNA molecule to
            another type (A --> C, etc). In the tables below, for both control and ESBL, one can observe that the
            <b>most frequent base change happens between AG and CT</b> and
            our variants consists of mainly SNP, with few insertion and deletion.
          </p>
          <figure>
            <img src="assets/img/overview.png" alt="" class="img-fluid">
          </figure>

          <h3>Possible Effects of Variants</h3>
          <p class="font-italic">The statistical data generated by snpEff illustrates that the <b>control and ESBL group
              are incredibly similar</b> with a few variance. For both groups, the <b>Missense/Silent ratio is about
              3</b> which is quite high.
            As this ratio increases, the more <b>mutations happening in our samples directly affect the
              amino acid it produces</b>. This could be reason for E. Coli bacteria gaining resistance to antibiotics.
            The rest of the statistics are similar and do not reveal how E. Coli bacterias become antibiotic resistant.
            Next, we will apply <b>snpSift filtration to reduce the scope</b> of the dataset to observe any noticeable
            changes.

          </p>

          <figure>
            <img src="assets/img/missense.png" alt="" class="img-fluid">
          </figure>

          <h3>Finding the Mutated Genes</h3>
          <p class="font-italic"> Using snpSift, only <b>high/moderate impact variants</b> (present in our ESBL samples
            and absent in at least one of our control samples) were kept. Originally, we had over 60 thousand variants,
            but only 50 remain after this filtration. Of those 50 variants, we extracted the <b>top 10 most frequently
              found
              mutated genes</b>. Below is a snippet of 5 variants that passed our filter.

          </p>

          <figure>
            <img src="assets/img/mutatedOne.png" alt="" class="img-fluid">
          </figure>

          <p class="font-italic">

            We can observe that in the remaining variants the most frequent mutations are from A -> G (see Base
            changes). In addition, the <b>variants_impact_MODIFIER</b> is quite high indicating that there are <b>more
              variances,
              with greater impact</b>, in this gene.

            <!-- ======= Results data Section ======= -->
          <section class="methodology">
            <div class="container">

              <div class="row">
                <div class="col-lg-6" data-aos="fade-right">
                  <div class="tab-content">
                    <div class="tab-pane active show" id="tab-5">
                      <figure>
                        <img src="assets/img/mutatedTwo.png" alt="" class="img-fluid">
                      </figure>
                    </div>
                    <div class="tab-pane" id="tab-6">
                      <figure>
                        <img src="assets/img/mutatedThree.png" alt="" class="img-fluid">
                      </figure>
                    </div>

                  </div>
                </div>
                <div class="col-lg-6 mt-4 mt-lg-0" data-aos="fade-left">
                  <ul class="nav nav-tabs flex-column">
                    <li class="nav-item">
                      <a class="nav-link active show" data-bs-toggle="tab" href="#tab-5">
                        <h4>Gene data</h4>
                        <p>This table contains the data of one of the top ten most commonly found mutated gene.</p>
                      </a>
                    </li>
                    <li class="nav-item mt-2">
                      <a class="nav-link" data-bs-toggle="tab" href="#tab-6">
                        <h4>Base changes</h4>
                        <p>This bar plot illustrates the base change happening in the remaining variants</p>
                      </a>
                    </li>
                  </ul>
                </div>
              </div>

            </div>
          </section><!-- End result-data Section -->
          <h3>GWAS results</h3>
          <p class="font-italic"> Here are the results of the GWAS Study done using logistic regression with Plink. The
            table is a <b>list of SNP’s with the lowest p-values</b>, indicating which variants are most likely to
            be statistically significant. <b>32,414 separate comparisons were made</b>, with <b>1,598 variants having an
              associated p-value of less than 0.05</b>. However, in studies dealing with biological data in particular,
            a
            technique called <a
              href="https://mathworld.wolfram.com/BonferroniCorrection.html#:~:text=The%20Bonferroni%20correction%20is%20a,the%20set%20of%20all%20comparisons)."
              target="_blank"> Bonferroni correction</a> is applied due to multiple comparisons being done
            simultaneously.
            It’s calculated by dividing the desired threshold, 0.05, by the number of comparisons, resulting in a much
            smaller threshold. After applying this correction, <b>0 of these variants</b> had p-values lower than this
            threshold.


          </p>

          <figure>
            <img src="assets/img/snp.png" alt="" class="img-fluid">
          </figure>
          <p class="font-italic">
            The <b>p-values</b> from the table were then visualized using a <a
              href="https://data.library.virginia.edu/understanding-q-q-plots/#:~:text=A%20Q%2DQ%20plot%20is%20a,truly%20come%20from%20Normal%20distributions."
              target="_blank">Q-Q
              plot</a> to check if their <b>distribution resembles
              data that is normally distributed</b>. From the plot shown below (see Q-Q plot), there appears to be a
            <b>skew of
              more
              extreme values</b>, indicated by the longer tails at the ends. A Manhattan plot was also produced to
            visualize
            how these p-values vary across the genome. In the Manhattan plot, <b>each dot represents a single SNP</b>,
            with
            the -log10 of
            the p-value on the y-axis and the genome location on the x-axis. <b>None of the SNPs hit the threshold</b>
            for
            Bonferroni correction, indicated by the horizontal red line at the top of the plot (see Manhattan plot).

          </p>
          <!-- ======= Results data Section ======= -->
          <section class="methodology">
            <div class="container">

              <div class="row">
                <div class="col-lg-6" data-aos="fade-right">
                  <div class="tab-content">
                    <div class="tab-pane active show" id="tab-7">
                      <figure>
                        <img src="assets/img/qq.png" alt="" class="img-fluid">
                      </figure>
                    </div>
                    <div class="tab-pane" id="tab-8">
                      <figure>
                        <img src="assets/img/manhattan.png" alt="" class="img-fluid">
                      </figure>
                    </div>

                  </div>
                </div>
                <div class="col-lg-6 mt-4 mt-lg-0" data-aos="fade-left">
                  <ul class="nav nav-tabs flex-column">
                    <li class="nav-item">
                      <a class="nav-link active show" data-bs-toggle="tab" href="#tab-7">
                        <h4>Q-Q Plot</h4>
                        <p>This Q-Q plot visualizes the p-values from the table</p>
                      </a>
                    </li>
                    <li class="nav-item mt-2">
                      <a class="nav-link" data-bs-toggle="tab" href="#tab-8">
                        <h4>Manhattan plot</h4>
                        <p>This Manhattan plot visualizes the variance of p-values across the genome</p>
                      </a>
                    </li>
                  </ul>
                </div>
              </div>

            </div>
          </section><!-- End result-data Section -->
    </section><!-- End results Section -->

    <!-- ======= Conclusion Section ======= -->
    <section id="conclusion" class="conclusion">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Conclusion</h2>

        </div>

        <div class="row">
          <p class="font-italic">Our study <b>aimed to explore and identify the genetic markers that characterize
            antibiotic resistance</b> in ESBL-producing E. Coli. The project involved comprehensive processing of our
              genomic data, followed by a genome wide association study and <b>analysis of found variants and their
                functional effects</b>. Through our analysis, we <b>noticed some observable differences</b> through principal component
            analysis and variant annotation, but the location of the specific SNPs responsible for ESBL production is
            <b>still unknown</b>.
            One of the <b>primary limitations</b> encountered was determining a definition for antibiotic resistance that could
            be <b>feasibly tested using a GWAS</b>. Although ESBL production has been proven to render E.Coli resistant to
            common antibiotics, there are many different physiological mechanisms that could also be used to classify
            other bacterial strains as antibiotic resistant. Another limitation was the limited sample size. This paired
            with the possible outliers from our principal component analysis could have given skewed results. If the
            study were to be repeated it would ideally involve hundreds, if not thousands of E.Coli samples. Time was
            another limiting factor, as a single sample takes approximately 60-70 minutes to be processed from raw reads
            to the Variant Call Format file needed for Plink and SnpEff. We also did not have access to a .gff file at
            the beginning of the study, which is an annotation file that is meant to be paired with the reference
            sequence used for alignment and variant calling. Because of this, we had to use two different reference
            genomes at different stages of the pipeline, which may have influenced the final GWAS and SnpEff outputs. In
            order to improve the logistic regression model, covariates such as %GC Content and more granular SNP
            filtering could be implemented.
            This study is a small stepping stone toward addressing the much broader problem of antibiotic resistance.
            The project pipeline can be applied to any strain of bacteria with a case and control group which can not
            only help identify significant genomic variants, but also their effects on protein transcription.
            Understanding these markers for antibiotic resistance can better inform scientists of how such physiological
            changes occur and what can be done to prevent the emergence of new antibiotic resistant strains in the
            future.


          </p>
          <figure>
            <img src="assets/img/limits.png" alt="" class="img-fluid">
          </figure>

          <p class="font-italic">
            What more can be done? Well, there are countless possibilities. The pipeline we created can be used, not
            just for E. Coli bacteria, but on any strain of bacteria. This can help combat other and future antibiotic
            resistant bacterias. Testing resistance to specific drugs can drive the development of new and effective
            treatments by pharmaceutical companies. A Machine learning model can apply our pipeline to more easily
            identify and predict whether a new bacterial strain has the potential to be antibiotic resistant. We can
            save countless of lives and prevent countless of illnesses. We believe that our study, this study, is simply
            just a stepping stone for something greater.
          </p>
        </div>

      </div>
    </section><!-- End conclusion section -->



    <!-- ======= Frequently Asked Questions Section ======= -->
    <section id="faq" class="faq">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Sources</h2>
        </div>

        <ul class="faq-list">

          <li data-aos="fade-up" data-aos-delay="100">
            <a data-bs-toggle="collapse" data-bs-target="#faq1" class="collapsed">Citations <i
                class="bx bx-chevron-down icon-show"></i><i class="bx bx-x icon-close"></i></a>
            <div id="faq1" class="collapse" data-bs-parent=".faq-list">
              <p>
                1. Center for Disease Control. “About Antibiotic Resistance.” Centers for Disease Control and
                Prevention, Centers for Disease Control and Prevention, 13 Mar. 2020,
                www.cdc.gov/drugresistance/about.html#:~:text=Antibiotic%20resistance%20happens%20when%20germs,and%20sometimes%20impossible%2C%20to%20treat.
              </p>
              <p>
                2. Cueni, Thomas B. “By 2050, Superbugs May Cost the Economy $100 Trillion.” IFPMA, International
                Federation of Pharmaceutical Manufacturers & Associations , 13 Nov. 2018,
                www.ifpma.org/global-health-matters/by-2050-superbugs-may-cost-the-economy-100-trillion/#:~:text=Antimicrobial%20resistance%20(AMR)%20is%20on,%E2%80%9Csuperbugs%E2%80%9D.
              </p>
              <p>
                3. Ikegawa, Shiro. “A short history of the genome-wide association study: where we were and where we are
                going.” Genomics & informatics vol. 10,4 (2012): 220-5. doi:10.5808/GI.2012.10.4.220
              </p>
              <p>
                4. Center for Disease Control. “ESBL-Producing Enterobacteriaceae.” Centers for Disease Control and
                Prevention, Centers for Disease Control and Prevention, 22 Nov. 2019,
                www.cdc.gov/hai/organisms/ESBL.html.
              </p>
              <p>
                5. Patel, IR. National Center for Biotechnology Information, U.S. National Library of Medicine, 22 May
                2015, trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA230969&o=acc_s%3Aa.
              </p>
              <p>
                6. Hokkaido University. National Center for Biotechnology Information, U.S. National Library of
                Medicine, 8 Dec. 2020, trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJDB10450&o=acc_s%3Aa.
              </p>
              <p>
                7. Babraham Institute. Babraham Bioinformatics - FastQC A Quality Control Tool for High Throughput
                Sequence Data, 26 Apr. 2010, www.bioinformatics.babraham.ac.uk/projects/fastqc/.
              </p>
              <p>
                8. Cutadapt — Cutadapt 3.1 documentation.https://cutadapt.readthedocs.io/en/stable/
              </p>
            </div>
          </li>
          <li data-aos="fade-up" data-aos-delay="200">
            <a data-bs-toggle="collapse" data-bs-target="#faq2" class="collapsed">Source Code <i
                class="bx bx-chevron-down icon-show"></i><i class="bx bx-x icon-close"></i></a>
            <div id="faq2" class="collapse" data-bs-parent=".faq-list">
              <p><a href="https://github.com/myhaider/antibiotic-resistance">Github Repository</a></p>
          </li>
        </ul>

      </div>
    </section><!-- End Frequently Asked Questions Section -->



  </main><!-- End #main -->

  <!-- ======= Footer ======= -->
  <footer id="footer">

    <div class="container footer-bottom clearfix">
      <div class="copyright">
        &copy; UCSD 2021
      </div>
      <div class="maintained">
        <p> This page is maintained by <a href="https://github.com/CaffeinatedCows">Alan Chen</a></p>
      </div>
  </footer><!-- End Footer -->

  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i
      class="bi bi-arrow-up-short"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>