<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta content="width=device-width, initial-scale=1.0" name="viewport">

  <title>Antibiotic Resistance</title>
  <meta content="" name="description">
  <meta content="" name="keywords">

  <!-- Google Fonts -->
  <link
    href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i|Montserrat:300,300i,400,400i,500,500i,600,600i,700,700i|Poppins:300,300i,400,400i,500,500i,600,600i,700,700i"
    rel="stylesheet">

  <!-- Vendor CSS Files -->
  <link href="assets/vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">
  <link href="assets/vendor/bootstrap-icons/bootstrap-icons.css" rel="stylesheet">
  <link href="assets/vendor/boxicons/css/boxicons.min.css" rel="stylesheet">

  <!-- Main CSS File -->
  <link href="assets/css/style.css" rel="stylesheet">

</head>

<body>
  <!-- ======= Welcome Section ======= -->
  <section id="welcome">
    <div class="welcome-container">
      <h1 data-aos="zoom-in">Antibiotic Resistance in E. Coli</h1>
      <h2 data-aos="fade-up">A Genome-wide study on Antibiotic Resistance in E. Coli</h2>
      <h2 data-aos="fade-up">
        <p>Alan Chen &emsp; Myra Haider &emsp; Jiayi Wu</p>
      </h2>
      <a data-aos="fade-up" data-aos-delay="200" href="#about" class="btn-get-started scrollto">Learn more</a>
    </div>
  </section><!-- End welcome -->

  <!-- ======= Header ======= -->
  <header id="header" class="d-flex align-items-center">
    <div class="container d-flex align-items-center justify-content-between">

      <nav id="navbar" class="navbar">
        <ul>
          <li><a class="nav-link scrollto active" href="#welcome">Home</a></li>
          <li><a class="nav-link scrollto" href="#about">About</a></li>
          <li><a class="nav-link scrollto" href="#methodology">Methodology</a></li>
          <li><a class="nav-link scrollto" href="#results">Results</a></li>
          <li><a class="nav-link scrollto" href="#results">Conclusion</a></li>
          <li><a class="nav-link scrollto" href="#faq">Sources</a></li>
        </ul>
        <i class="bi bi-list mobile-nav-toggle"></i>
      </nav><!-- .navbar -->

    </div>
  </header><!-- End Header -->

  <main id="main">

    <!-- ======= About Section ======= -->
    <section id="about" class="about">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Abstract</h2>
          <p> One of the greatest challenges in public health is the growing number of bacterial species that have
            developed resistance to antibiotics through point mutations in their genome.
            Our project aims to identify these genetic markers of antibiotic resistance through a genome wide
            association study between 36 antibiotic resistant E.Coli samples and 36 control samples. Genetic variants
            will be
            checked for statistical significance, and analyzed for any functional effects.
          </p>
        </div>

        <div class="row">
          <div class="col-lg-6" data-aos="fade-right">
            <div class="image">
              <img src="assets/img/antibiotic.png" class="img-fluid" alt="antibiotic resistance">
            </div>
          </div>
          <div class="col-lg-6" data-aos="fade-left">
            <div class="content pt-4 pt-lg-0 pl-0 pl-lg-3 ">
              <h3>"By 2050, superbugs could kill 10 million people a year"</h3>
              <p class="font-italic">
                Although <b>antibiotic resistant bacteria</b> have been studied for decades, their whole genome
                sequencing has started relatively recently <sup>[3]</sup>. Due to their immunity to known
                medicine, their exposure to humans can lead to infections that are currently <b>impossible to cure</b>
                <sup>[1]</sup>.
                According to the International Federation of Pharmaceutical
                Manufacturers & Associations
                (IFPMA), antibiotic resistance will:
              </p>
              <ul>
                <li><i class="bx bx-check-double"></i> <b>Force 28 million more people into poverty<sup>[2]</sup> </b>
                </li>
                <li><i class="bx bx-check-double"></i> <b>Cost the global economy up to 100 trillion
                    dollars<sup>[2]</sup></b></li>
                <li><i class="bx bx-check-double"></i> <b>Kill up to 10 million people a year by 2050<sup>[2]</sup></b>
                </li>
              </ul>
              <p>
                This study focuses on <b>Enterobacteriaceae</b>, specifically E. Coli, a common cause of infections,
                both
                in healthcare
                settings and communities <sup>[4]</sup>. Certain strains of E. Coli have developed an especially
                dangerous
                resistance mechanism, the ability to produce an enzyme known as <b>extended-spectrum
                  beta-lactamase</b>, or ESBL <sup>[4]</sup>. ESBL is capable of breaking down multiple types of
                antibiotics
                such as penicillin, <b>rendering them ineffective</b> <sup>[4]</sup>.
              </p>
              <p>
                Understanding the genetic markers of antibiotic resistance can lead to better, <b>more effective
                  treatments</b> to combat the growing concern of superbugs.
              </p>
            </div>
          </div>
        </div>
      </div>
    </section><!-- End About Us Section -->

    <!-- ======= Methodology Section ======= -->
    <section id="methodology" class="methodology">
      <div class="container">
        <div class="section-title" data-aos="fade-up">
          <h2>Metholodgy</h2>
        </div>
        <div class="row">
          <h4 id="pipeline">An Overview of the Pipeline</h4>

          <figure>
            <img src="assets/img/pipeline.png" class="img-fluid" alt="pipeline">
          </figure>
          <h3>The Data</h3>
          <p class="font-italic">
            The study was conducted using two groups of samples, one with an <b>ESBL-enzyme
              producing</b> strain of E.Coli (denoted with DRR as the suffix), and a non-antibiotic resistant <b>control
              group</b> <sup>[5][6]</sup>. For each group, <b>sample sizes were 36 whole genomes</b>. The
            samples were obtained from <a
              href="https://www.ncbi.nlm.nih.gov/Traces/study/?acc=SRP058582&f=librarylayout_s%3An%3Apaired%3Bbases_l%3An%3A0%253A999999999%3Binstrument_s%3An%3Aillumina%2520miseq%3Ac&o=acc_s%3Aa&s=SRR10001251,SRR10018351,SRR10001259,SRR10001268,SRR10012673,SRR10012674,SRR10012676,SRR10012677,SRR10012698,SRR10012702,SRR10012704,SRR10012906,SRR10012913,SRR10013003,SRR10013021,SRR10013010,SRR10013022,SRR10013025,SRR10013029,SRR10013030,SRR10013032,SRR10013114,SRR10013117,SRR10013127,SRR10013130,SRR10013322,SRR10013157,SRR10013348,SRR10013353,SRR10013371,SRR10013397,SRR10013400,SRR10013405,SRR10013432,SRR10013439,SRR10013442">Illumina
              NCBI's SRA Run Selector</a> and formatted as <b>.fastq files</b> <sup>[5][6]</sup>. Each sample has two
            files associated with it, as they are paired-end reads, totalling to 144 files.
          </p>
          <h3>Quality Control</h3>
          <p>To ensure these 144 files will yield valid results later in the analysis, each was put through a quality
            control software called FastQC. <b>FastQC</b> takes in a single .fastq file and <b>outputs a summary
              report</b> listing
            basic statistics about the reads, including quality scores from base calling, the presence of adapter
            sequences, and percent N content, which indicates missing data. </p>
          <p>While it’s important to check for quality of
            each sample individually, the FastQC results of our <b>case and control groups were also aggregated</b> in
            order to
            <b>identify</b> any major <b>differences in quality</b> between them that could affect the GWAS results.
            <b>All 144
              files passed</b> the default FastQC check for basic statistics and adapter content, thus they were all
            included in the study.
            However, the <b>antibiotic resistance group</b> had more samples that had an <b>unusually high per
              sequence GC content</b>, indicated by the higher number of ESBL_Samples that failed this check (see <a
              href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/5%20Per%20Sequence%20GC%20Content.html"
              target="_blank">GC
            </a> and <a
              href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/10%20Adapter%20Content.html"
              target="_blank">Adapter Content</a>). This could be due to either the data being lower quality, OR this
            possibly being an actual
            physiological marker for genomes in this group.
          </p>
          <div class="col-lg-6" data-aos="fade-right">

            <div class="tab-content">
              <div class="tab-pane active show" id="tab-1">
                <figure>
                  <img src="assets\img\quality score.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-2">
                <figure>
                  <img src="assets/img/sampleReport.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-3">
                <figure>
                  <img src="assets/img/basicstats.png" alt="" class="img-fluid">
                </figure>
              </div>
              <div class="tab-pane" id="tab-4">
                <figure>
                  <img src="assets/img/perGCadapter.png" alt="" class="img-fluid">
                </figure>
              </div>
            </div>
          </div>
          <div class="col-lg-6 mt-4 mt-lg-0" data-aos="fade-left">
            <ul class="nav nav-tabs flex-column">
              <li class="nav-item">
                <a class="nav-link active show" data-bs-toggle="tab" href="#tab-1">
                  <h4>Quality Score</h4>
                  <p>The distribution of quality scores across reads of different lengths</p>
                </a>
              </li>
              <li class="nav-item mt-2">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-2">
                  <h4>Summary Report</h4>
                  <p>Table of summary statistics for a sample</p>
                </a>
              </li>
              <li class="nav-item mt-3">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-3">
                  <h4>Basic Statistics</h4>
                  <p>The barchart depicts the total number of samples that passed the basic statistics test between
                    groups</p>
                </a>
              </li>
              <li class="nav-item mt-4">
                <a class="nav-link" data-bs-toggle="tab" href="#tab-4">
                  <h4>GC and Adapter Content</h4>
                  <p>The barchart depicts the total number of samples that passed the basic statistics test between
                    groups</p>
                </a>
              </li>
            </ul>
          </div>
          <h3>Sequence Alignment, File Conversion, Sorting</h3>
          <p class="font-italic">
            A pair of .fastq files for one sample contain the raw reads from the entire genome, but those reads are
            essentially small fragments that need to be put together like a puzzle. This is where the process of
            alignment comes in. Using a previously known E. Coli reference genome, we utilized an alignment software
            called
            <b>Bowtie2</b> which algorithmically pieces together these <b>reads into a contiguous sequence</b>. The
            output is a file
            in sam format, which is a text readable version of this sequence. Using <a href="http://www.htslib.org/"
              target="_blank">samtools</a> and <a href="https://broadinstitute.github.io/picard/"
              target="_blank">picard</a>, these sam files were then converted to
            .bam - a compressed version that runs more efficiently through bioinformatics software.
          </p>
          <figure>
            <img src="assets/img/alignment.png" alt="" class="img-fluid">
          </figure>
          <h3>Variant Calling</h3>
          <p class="font-italic">
            The 72 cleaned and formatted .bam files then undergo Variant Calling, which is the process of identifying
            the <a
              href="https://isogg.org/wiki/Single-nucleotide_polymorphism#:~:text=A%20single%2Dnucleotide%20polymorphism%20(SNP,paired%20chromosomes%20in%20an%20individual."
              target="_blank">Single Nucleotide Polymorphisms</a>, or SNP’s in each sample. These are the point
            mutations that can
            differentiate samples, with the goal being able to find the SNP’s that are significantly different between
            our antibiotic-resistant and control groups. This was done using GATK’s <a
              href="https://gatk.broadinstitute.org/hc/en-us/articles/360037225632-HaplotypeCaller"
              target="_blank">HaplotypeCaller</a> tool, which outputs
            a vcf that contains the list of variants found in a single sample. All of these vcf files were then merged
            into a single file using <a href="http://samtools.github.io/bcftools/bcftools.html"
              target="_blank">bcftools</a> which can be used for our GWAS.
          </p>
          <figure>
            <img src="assets/img/variant.png" alt="" class="img-fluid" style="padding-bottom:30px">
          </figure>
        </div>

      </div>
    </section><!-- End data Section -->

    <!-- ======= Results Section ======= -->
    <section id="results" class="methodology">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Results</h2>
        </div>

        <div class="row">
          <h3>PCA Analysis</h3>
          <p class="font-italic">So that was our entire process of data setup. We can move onto the actual “analysis” of
            these genomic variants, starting with a basic PCA to determine if there is any meaningful difference between
            the Antibiotic-resistant and Control groups. The principle components were calculated using Plink, and the
            scatterplot of the first two PC’s are shown on the right. The initial results look promising, with most of
            the antibiotic-resistant E. Coli samples being clustered together on the right. The control samples seem to
            have more variance, especially the three outliers in the top left. Because the sizes of our sample groups
            were already quite small, we chose to keep them in the study.
          </p>
          <figure>
            <img src="assets/img/pca.png" alt="" class="img-fluid">
          </figure>
          <h3>Genome Wide Association Study</h3>
          <p class="font-italic">In order to perform the GWAS, we used a Logistic Regression model which compares the
            two groups and calculates a p-value for each SNP. The model was further tuned by using the first two
            principal components as covariates, and all SNP’s with a genotyping rate of less than 60% were removed from
            the analysis. The genotyping rate was used as a measure of missing data, and we found that 60% was the best
            threshold to use in order to minimize the skew of our results.

          </p>
          <figure>
            <img src="assets/img/gwas.png" alt="" class="img-fluid">
          </figure>
          <h3>Variant Annotation</h3>
          <p class="font-italic">To perform further analysis on our variants, we chose snpEff to help us with the
            variant annotation. We used the E-coli database that snpEff provided us as the reference in our annotation.
            This step basically gives us an annotated vcf file, a succinct summary report and gene_name table. We can
            then use snpSift to filter out the common variants which presented in both group and leave only the variants
            we need. This step helps us reduce the scope significantly and enable us to do more specific analyzation
            with it.

          </p>
          <figure>
            <img src="assets/img/annotation.png" alt="" class="img-fluid">
          </figure>
          <h3>snpEff Summary</h3>
          <p class="font-italic">To briefly discuss about the summary table we got. Since the number of errors we
            encountered is relatively small compared with our datasize, I want to focused more on the warnings we got.
            We can see that there is a bunch numbers of warnings show in both control and esbl groups. These warnings
            mainly consist of ref_dose_not match_genome error. I think it is because we used the presetted database of
            snpEff. And it may be better if we can build our own database next time. We can also see that the variant
            rate of the esbl group is much higher than the control group. Which is a good point indicates that there may
            be some interesting finding in further analysis. We will just skip the other lines of these table since they
            are just ordinary statistics with no specific interests.


          </p>
          <figure>
            <img src="assets/img/snpeff.png" alt="" class="img-fluid">
          </figure>

          <h3>Overview of Variants</h3>
          <p class="font-italic">So here is an brief overview of the base change and variants type of our two groups. So
            the Base change is a basic gene level mutation which change the original gene base of a DNA molecule to
            another type of it.We can see that the base changes frequently happens as Transitions between AG and CT. And
            our variants are consist of mainly SNP, but also some insertion and deletion.



          </p>
          <figure>
            <img src="assets/img/overview.png" alt="" class="img-fluid">
          </figure>
        </div>

      </div>
      <!-- ======= Results data Section ======= -->
      <section class="methodology">
        <div class="container">

          <div class="row">
            <div class="col-lg-6" data-aos="fade-right">
              <div class="tab-content">
                <div class="tab-pane active show" id="tab-1">
                  <figure>
                    <img src="assets/img/image.png" alt="" class="img-fluid">
                  </figure>
                </div>
                <div class="tab-pane" id="tab-2">
                  <figure>
                    <img src="assets/img/image.png" alt="" class="img-fluid">
                  </figure>
                </div>
                <div class="tab-pane" id="tab-3">
                  <figure>
                    <img src="assets/img/image.png" alt="" class="img-fluid">
                  </figure>
                </div>
                <div class="tab-pane" id="tab-4">
                  <figure>
                    <img src="assets/img/image.png" alt="" class="img-fluid">
                  </figure>
                </div>
              </div>
            </div>
            <div class="col-lg-6 mt-4 mt-lg-0" data-aos="fade-left">
              <ul class="nav nav-tabs flex-column">
                <li class="nav-item">
                  <a class="nav-link active show" data-bs-toggle="tab" href="#tab-5">
                    <h4>Results 1</h4>
                    <p>How this result is obtained...</p>
                  </a>
                </li>
                <li class="nav-item mt-2">
                  <a class="nav-link" data-bs-toggle="tab" href="#tab-6">
                    <h4>Results 2</h4>
                    <p>How this result is obtained...</p>
                  </a>
                </li>
                <li class="nav-item mt-2">
                  <a class="nav-link" data-bs-toggle="tab" href="#tab-7">
                    <h4>Results 3</h4>
                    <p>How this result is obtained...</p>
                  </a>
                </li>
                <li class="nav-item mt-2">
                  <a class="nav-link" data-bs-toggle="tab" href="#tab-8">
                    <h4>Results 4</h4>
                    <p>How this result is obtained...</p>
                  </a>
                </li>
              </ul>
            </div>
          </div>

        </div>
      </section><!-- End result-data Section -->
    </section><!-- End results Section -->

    <!-- ======= Conclusion Section ======= -->
    <section id="conclusion" class="conclusion">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Conclusion</h2>
          <p> This section contains our conclusion...
          </p>
        </div>

        <div class="row">
          <div class="col-lg-6" data-aos="fade-left">
            <div class="content pt-4 pt-lg-0 pl-0 pl-lg-3 ">
              <p class="font-italic">
                Conclusion is that ....
              </p>

            </div>
          </div>
        </div>

      </div>
    </section><!-- End conclusion section -->



    <!-- ======= Frequently Asked Questions Section ======= -->
    <section id="faq" class="faq">
      <div class="container">

        <div class="section-title" data-aos="fade-up">
          <h2>Sources</h2>
        </div>

        <ul class="faq-list">

          <li data-aos="fade-up" data-aos-delay="100">
            <a data-bs-toggle="collapse" data-bs-target="#faq1" class="collapsed">Citations <i
                class="bx bx-chevron-down icon-show"></i><i class="bx bx-x icon-close"></i></a>
            <div id="faq1" class="collapse" data-bs-parent=".faq-list">
              <p>
                1. Center for Disease Control. “About Antibiotic Resistance.” Centers for Disease Control and
                Prevention, Centers for Disease Control and Prevention, 13 Mar. 2020,
                www.cdc.gov/drugresistance/about.html#:~:text=Antibiotic%20resistance%20happens%20when%20germs,and%20sometimes%20impossible%2C%20to%20treat.
              </p>
              <p>
                2. Cueni, Thomas B. “By 2050, Superbugs May Cost the Economy $100 Trillion.” IFPMA, International
                Federation of Pharmaceutical Manufacturers & Associations , 13 Nov. 2018,
                www.ifpma.org/global-health-matters/by-2050-superbugs-may-cost-the-economy-100-trillion/#:~:text=Antimicrobial%20resistance%20(AMR)%20is%20on,%E2%80%9Csuperbugs%E2%80%9D.
              </p>
              <p>
                3. Ikegawa, Shiro. “A short history of the genome-wide association study: where we were and where we are
                going.” Genomics & informatics vol. 10,4 (2012): 220-5. doi:10.5808/GI.2012.10.4.220
              </p>
              <p>
                4. Center for Disease Control. “ESBL-Producing Enterobacteriaceae.” Centers for Disease Control and
                Prevention, Centers for Disease Control and Prevention, 22 Nov. 2019,
                www.cdc.gov/hai/organisms/ESBL.html.
              </p>
              <p>
                5. Patel, IR. National Center for Biotechnology Information, U.S. National Library of Medicine, 22 May
                2015, trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA230969&o=acc_s%3Aa.
              </p>
              <p>
                6. Hokkaido University. National Center for Biotechnology Information, U.S. National Library of
                Medicine, 8 Dec. 2020, trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJDB10450&o=acc_s%3Aa.
              </p>
            </div>
          </li>
          <li data-aos="fade-up" data-aos-delay="200">
            <a data-bs-toggle="collapse" data-bs-target="#faq2" class="collapsed">Source Code <i
                class="bx bx-chevron-down icon-show"></i><i class="bx bx-x icon-close"></i></a>
            <div id="faq2" class="collapse" data-bs-parent=".faq-list">
              <p><a href="https://github.com/myhaider/antibiotic-resistance">Github Repository</a></p>
          </li>
        </ul>

      </div>
    </section><!-- End Frequently Asked Questions Section -->



  </main><!-- End #main -->

  <!-- ======= Footer ======= -->
  <footer id="footer">

    <div class="container footer-bottom clearfix">
      <div class="copyright">
        &copy; UCSD 2021
      </div>
      <div class="maintained">
        <p> This page is maintained by <a href="https://github.com/CaffeinatedCows">Alan Chen</a></p>
      </div>
  </footer><!-- End Footer -->

  <a href="#" class="back-to-top d-flex align-items-center justify-content-center"><i
      class="bi bi-arrow-up-short"></i></a>

  <!-- Vendor JS Files -->
  <script src="assets/vendor/aos/aos.js"></script>
  <script src="assets/vendor/bootstrap/js/bootstrap.bundle.min.js"></script>
  <script src="assets/vendor/glightbox/js/glightbox.min.js"></script>
  <script src="assets/vendor/isotope-layout/isotope.pkgd.min.js"></script>
  <script src="assets/vendor/php-email-form/validate.js"></script>
  <script src="assets/vendor/swiper/swiper-bundle.min.js"></script>

  <!-- Template Main JS File -->
  <script src="assets/js/main.js"></script>

</body>

</html>